spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/BindingBeansRegistrar.java

```diff
@@ -37,11 +37,11 @@
 	@Override
 	public void registerBeanDefinitions(AnnotationMetadata metadata,
 			BeanDefinitionRegistry registry) {
-		MultiValueMap<String, Object> attributes = metadata.getAllAnnotationAttributes(
-				EnableModule.class.getName(), false);
-		List<String> registeredChannelNames = new ArrayList<>();
+		MultiValueMap<String, Object> attributes = metadata.getAllAnnotationAttributes(EnableBinding.class.getName(),
+				false);
 		for (Class<?> type : collectClasses(attributes.get("value"))) {
-			BindingBeanDefinitionRegistryUtils.registerChannelBeanDefinitions(type, type.getName(), registry);
+			BindingBeanDefinitionRegistryUtils.registerChannelBeanDefinitions(type,
+					type.getName(), registry);
```

Comment:
Something is weird with your IDE formatter ;) I notice the code format changes in few places...